tests/phpunit/CRM/Core/BAO/FinancialTrxnTest.php

```diff
@@ -244,4 +244,14 @@ public function testUpdateCreditCardDetails() {
     $this->assertEquals($financialTrxn['pan_truncation'], 4567);
   }
 
+  /**
+   * Test CRM_Core_BAO_FinancialTrxn::formatCreditCardDetails(..)
+   */
+  public function testFormatCreditCardDetails() {
```

Comment:
I'm kind of hopeful the formatCreditCardDetails function might be dead by the time this is merged